src/Rewards.js

```diff
@@ -2,21 +2,19 @@ import { usePostTransaction } from "./mutations";
 import { Value } from "./helpers";
 import { ELC } from "./constants";
 import { useGetIssuanceRewards } from "./queries";
-import React from "react";
-import useSound from "use-sound";
+import { default as React, useRef } from "react";
```

Comment:
Is there a functional implication of using React and then running React.useState or is this your preference for being explicit about where the methods are coming from? In other words, why not do:

```javascript
import { useState, useRef } from "react";
```

You obviously have your reasons. I want to know what they are! This is a comment about the usage throughout the codebase.

Reply:
Ah, no reason actually! Thanks for the nudge! -> [Explicity import React exports](https://github.com/ellipticoin/ellipticoin-wallet/pull/21/commits/f6d65b9e101615d429130237a4bc1b6dd7ff7032)